packages/valory/contracts/service_registry/contract.py

```diff
@@ -21,20 +21,24 @@
 
 import hashlib
 import logging
-from typing import Any, Dict, Tuple, cast
+from typing import Any, Dict, Tuple, Union, cast
 
 from aea.configurations.base import PublicId
 from aea.contracts.base import Contract
 from aea_ledger_ethereum import EthereumApi, LedgerApi
 
 
-DEPLOYED_BYTECODE_MD5_HASH = "288a5ecfe0b685d93f174ec24f743458679a4948d6517d77957fd67b72e5982eb85f543329059eabada38428eef9d548e407b1e0d60dd83af60099ea76bf5a76"
+# test_contract and test_agents, respectively...
+DEPLOYED_BYTECODE_MD5_HASH = "75d2a79df580ba1353211f93c479a9d6b78cc8f14e724290329e274fdcab4dc8cc04adbd8efbbce00a01af2dd6873f7e66a8c338a3d4f87a31ab0e283eef89de"
 
```

Comment:
Why do we need this here? Testing purposes?

Reply:
@Karrenbelt

Reply:
Opened #999.